content/wip-cards/1_card/contents.lr

```diff
@@ -0,0 +1,14 @@
+work: Ephesians Series
+---
+attribution: Gary DeLashmutt    
+---
+link: https://teachings.dwellcc.org/?SeriesID=109&author%5B%5D=2
+---
+icon_slug: fa-files-o
+---
+body: The _quick_ brown **fox** jumped `over` the lazy dog.
+---
+languages:
+French
+Russian
+Spanish
```

Comment:
I removed the translator's names until we could confirm there's no security issue.  I don't know that there is, just wanted to confirm.